ako-operator/helm/ako-operator/values.yaml

```diff
@@ -89,6 +90,8 @@ resources:
     cpu: 100m
     memory: 200Mi
 
+podSecurityContext: {}
```

Comment:
We don't need this here. If this was meant for AKO, we already add it via the AKO operator code.